common/src/main/java/com/redhat/cloud/notifications/models/dto/v1/endpoint/properties/CamelPropertiesDTO.java

```diff
@@ -0,0 +1,67 @@
+package com.redhat.cloud.notifications.models.dto.v1.endpoint.properties;
+
+import com.redhat.cloud.notifications.models.dto.v1.endpoint.properties.secrets.BasicAuthenticationDTO;
+import com.redhat.cloud.notifications.models.validation.ValidNonPrivateUrl;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+
+import java.util.Map;
+
+public final class CamelPropertiesDTO extends EndpointPropertiesDTO {
+    private Boolean disableSslVerification = Boolean.FALSE;
```

Comment:
Any reason why `@NotNull` was removed from the entity but not added here?

Reply:
Added!